<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkStructFile</class>
 <widget class="QDialog" name="WorkStructFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>631</width>
     <height>452</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Население:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit_3"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEdit_4"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="textEdit_5"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Площадь города:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Первая буква страны:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Является ли столицей:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Название 5 улиц:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Кол-во домов на улицах:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="textEdit_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="inputFileButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>475</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Записать в файл</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Содержимое файла:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>391</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ПРОСМОТРЕТЬ</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>80</y>
     <width>391</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
